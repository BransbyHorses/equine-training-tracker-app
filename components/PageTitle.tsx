import React from 'react';
import { Typography, Box, Paper } from '@mui/material';

export default function PageTitle(props: any) {
    return (
            <Box
                px={4}
                py={2} 
                sx={{ display: 'flex', justifyContent: 'space-between' }}>
                <Typography
                    variant="h4"
                    color="primary"
                    sx={{
                        fontSize: '24px',
                        fontWeight: '700',
                        color: 'common.black'
                    }}
                >
                    {props.title}
                </Typography>
            </Box>
    );
}
