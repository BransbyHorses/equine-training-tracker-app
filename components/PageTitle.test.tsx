import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import PageTitle from "./PageTitle";

describe("PageTitle", () => {
	it("renders with title", () => {
		render(<PageTitle title="Test Page Title" />);
		const pageTitle = screen.getByText("Test Page Title");
		expect(pageTitle).toBeInTheDocument();
		expect(pageTitle).toMatchSnapshot();
	});
});
