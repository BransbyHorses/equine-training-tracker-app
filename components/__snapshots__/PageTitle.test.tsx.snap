// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle renders with title 1`] = `
<h4
  class="MuiTypography-root MuiTypography-h4 css-1lni2c2-MuiTypography-root"
>
  Test Page Title
</h4>
`;
